src/app/game-state/followers.service.ts

```diff
@@ -88,18 +92,26 @@ export class FollowersService {
         if (this.characterService.characterState.equipment.head && 
           this.characterService.characterState.equipment.head.armorStats){
           this.characterService.characterState.equipment.head.armorStats.durability += follower.power;
+          this.characterService.characterState.equipment.head.armorStats.defense += Math.floor(follower.power/10);
```

Comment:
This section ended up getting really bulky. Can we just loop through the slots so we have less duplicated code?

Reply:
I'll try doing that, have to wrap my head around the specific logic of how.

Reply:
Okay, so I want to do this, but I have no idea how to loop the slot since it's part of characterService and not inventoryService.